velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -266,12 +270,24 @@ HiveConnectorTestBase::makeHiveInsertTableHandle(
   VELOX_CHECK_EQ(numPartitionColumns, partitionedBy.size());
   VELOX_CHECK_EQ(numBucketColumns, bucketedBy.size());
   VELOX_CHECK_EQ(numSortingColumns, sortedBy.size());
+
+  std::unique_ptr<dwio::common::FlushPolicy> flushPolicy = nullptr;
+  if (tableStorageFormat == dwio::common::FileFormat::DWRF) {
+    flushPolicy =
+        std::make_unique<velox::dwrf::LambdaFlushPolicy>([]() { return true; });
+  } else if (tableStorageFormat == dwio::common::FileFormat::PARQUET) {
+#ifdef VELOX_ENABLE_PARQUET
```

Comment:
Do we have a test that executes the Parquet path?

Reply:
It seems not. Removed parquet test.